qutebrowser/config/config.py

```diff
@@ -731,28 +732,49 @@ def set_command(self, win_id, section_=None, option=None, value=None,
             print_ = True
         else:
             with self._handle_config_error():
-                if option.endswith('!') and option != '!' and value is None:
+                if option.endswith('!') and option != '!' and len(values) == 0:
```

Comment:
Would that be considered better style? As you pointed out in another PR, relying on “falsiness” when an equivalent but explicit/clear check is only minutely longer seems like it hurts readability more than it helps it.